crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -531,7 +531,7 @@ impl SlidingSync {
     /// Send a sliding sync request.
     ///
     /// This method contains the sending logic. It takes a generic `Request`
-    /// because it can be a Simplified MSC3575 or a MSC3575 `Request`.
+    /// because it can be an MSC4186 or an MSC3575 `Request`.
```

Comment:
I'm not a native English person, so bear with me, but is it correct to say `an MSC request` instead of `a MSC request`?

Reply:
"an MSC request" when you are expecting (as I am) that the reader will pronounce it "em ess see" when they read.